src/envo/envo-edit.owl

```diff
@@ -11198,12 +11198,21 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000481> ObjectSomeValuesFrom(<
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00000482> (sea floor)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "MA:ma"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000482> "The ground surface that lies beneath a sea."^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Seabed>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "MA:ma"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000482> "A marine bed which is covered by marine sediments."^^xsd:string)
```

Comment:
>A marine bed which is covered by marine sediments.

Not sure about this. Isn't the bed boundary defined by the sediments? I suppose the confusion comes from the use of the ambiguous word "ground" in[ submerged bed](http://purl.obolibrary.org/obo/ENVO_00000501).

And there is [a difference between oceans and seas](https://oceanservice.noaa.gov/facts/oceanorsea.html) which supports the use of both sea floor and ocean floor.